01-getting-started/003-ecr-setup.md

```diff
@@ -45,7 +78,18 @@ resource "kubernetes_secret" "ecr-repo" {
 
 This will create an image repository at `<account_number>.dkr.ecr.eu-west-1.amazonaws.com/my-team-name/my-app-name` along with the credentials to push to it. Make sure you adjust the values of `team_name`, `repo_name` and `namespace` to what is appropriate for your environment.
 
-Commit the changes and raise a pull request. Once merged, our pipeline will apply the changes and create the AWS resources.
+5\. git add, commit and push to your branch.
+
+```bash
+
+  $ git add main.tf
+
+  $ git commit -m "created Terraform module for ECR"
+
+  $ git push
+
+```
+6\. Once your request has been approved and the branches are merged, it will trigger our build pipeline which will apply the Terraform module and create the resources.
 
```

Comment:
Can we add some instructions on how to get your secret from kubernetes once the terraform has run? People often ask us for the credentials in #cloud-platform-users, so I think they are not getting that you can get the secret from the cluster.

Reply:
it's in there'Accessing the credentials'